### ISSUE TITLE
The aria-label of the ComboBox is not applied on the correct DOM element

### ISSUE THREAD

**Issue by PekoPPT** [telerik/kendo-vue#432]
**Describe the bug**
When defined, the **aria-label** of the ComboBox is applied to the wrapper element with a "**k-combobox**" class while it should be applied to the input element with the "**k-input-inner**" class

**To Reproduce**
1. Open this StackBlitz example
2. Open browser's Devtools and inspect the ComboBox element
3. See where the **aria-label** is applied

**Expected behavior**
The **aria-label** should be applied to the inner **input** element

**Screenshots**
<img width="1204" alt="image" src="https://github.com/telerik/kendo-vue/assets/41293735/502667f3-e792-4c0d-8cdb-6f17e2e5aebf">

**Comment by PekoPPT** [comment]
The issue is fixed in version 3.13 of the Kendo UI for Vue Native components.
